module/VuFind/tests/unit-tests/src/VuFindTest/ILS/Driver/FolioTest.php

```diff
@@ -170,9 +170,13 @@ protected function createConnector(string $test, ?array $config = null): void
             $manager = new \Laminas\Session\SessionManager();
             return new \Laminas\Session\Container("Folio_$namespace", $manager);
         };
+        // Create a mock Auth Manager
+        $authManager = $this->getMockBuilder(\VuFind\Auth\Manager::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
If you take my above suggestion, you won't need this at all, but if you decide to keep it you can simplify this to:

```suggestion
        $authManager = $this->createMock(\VuFind\Auth\Manager::class);
```